src/com/facebook/buck/android/AbstractAndroidGraphEnhancementResult.java

```diff
@@ -59,7 +59,7 @@
 
   @Value.Derived
   default Optional<PreDexMerge> getPreDexMerge() {
-    return getDexMergeRule().transform(left -> Optional.of(left), right -> Optional.empty());
+    return getDexMergeRule().transform(left -> Optional.of(left), right -> Optional.<PreDexMerge>empty());
```

Comment:
revert?

Reply:
this fixes that compile error you noted in a buck issue a while back, do we not want it?

Reply:
put it in a separate change I guess